utils/format-modules.js

```diff
@@ -72,7 +72,7 @@ function groupModules(bundle) {
       });
     }),
     _.orderBy(_.get("size.minGz"), "desc")
-  )(bundle.metrics.sizes);
+  )(realBundleMatch.metrics ? bundle.metrics.sizes : []);
```

Comment:
`realBundleMatch` is undefined and failing lint. Has this code been tested yet?

Reply:
Sorry, it seems I get typo :cry: . Going to fix it in few minutes using my phone.